src/overlays/actors/ovl_En_Elf/z_en_elf.c

```diff
@@ -675,8 +675,8 @@ void func_8088DD34(EnElf* this, PlayState* play) {
         !func_8088C804(&this->actor.world.pos, &refActor->actor.world.pos, 10.0f)) {
         Health_ChangeBy(play, 0x80);
         if (this->fairyFlags & 0x200) {
-            Parameter_AddMagic(play, ((void)0, gSaveContext.unk_3F30) +
-                                         (gSaveContext.save.playerData.doubleMagic * 0x30) + 0x30);
+            Magic_Add(play, ((void)0, gSaveContext.magicFillTarget) +
+                                (gSaveContext.save.playerData.isDoubleMagicAcquired + 1) * MAGIC_NORMAL_METER);
```

Comment:
Just documenting this here for reference, maybe this line deserves a macro:

This line shows up over and over again and seems equivalent to OoT's `Magic_Fill(play)` function, which was removed in MM. So if MM want's to fill the magic meter to full, this line here how it's called.

The `(gSaveContext.save.playerData.isDoubleMagicAcquired + 1) * MAGIC_NORMAL_METER` part should already be equal to the current magic capacity. So 
`Magic_Add((gSaveContext.save.playerData.isDoubleMagicAcquired + 1) * MAGIC_NORMAL_METER` 
alone should already add enough magic that it's guaranteed to fill to capacity. 

Adding on `magicFillTarget` to that is strange as this doesn't really do anything as adding more to a already full magic refill will just lead to a full magic refill, and it's unclear as to the logic as to what `magicFillTarget` is expected to be when calling this function in normal gameplay.

Reply:
I would tend to agree. Looks like it is used often enough in this PR alone.